conf/main/mindmaps-engine.properties

```diff
@@ -48,4 +48,4 @@ backgroundTasks.time-lapse=60000
 loader.threads=0
 
 #MindmapsDB version
-project.version=0.1.0
+project.version=0.2.0-SNAPSHOT
```

Comment:
Yes, I'm using it in GraqlShell to print the version. The relevant stuff to look at is `mindmaps-graql-shell/src/main/java-templates` and the `templating-maven-plugin` in the pom.

Perhaps this version class should go in `mindmaps-base`, so we can both access it?